wmake.ps1

```diff
@@ -19,19 +19,44 @@ Param(
     [Parameter(Position=0, 
     HelpMessage="Enter the build target")]
     [Alias("target")]
-    [AllowEmptyString()]
     [ValidateSet("all", "clean", "test", "erigon","rpcdaemon","rpctest", "hack", "state", "integration", "db-tools", "sentry")]
-    [string]$BuildTarget="erigon"
+    [string[]]$BuildTargets=@("erigon","rpcdaemon")
```

Comment:
let's add sentry and integration?

Reply:
You mean as defaults ? (i.e. if no target is user provided ?)